### ISSUE TITLE
Consider adding support for medications that are not used every day

### ISSUE THREAD

**Issue by DeckOfPandas** [takeyourmeds/takeyourmeds-web#101]
Off the top of my head: methotrexate, olendronate, pain patches (can you tell I've been doing orthopaedics and rheumatology recently)

**Comment by pacharanero** [comment]
methotrexate: weekly
zolendronic acid: monthly
alendronic acid: weekly if it's the 70mg dose (daily if it's the 10mg dose)
buprenorphine patches: change weekly
fentanyl patches: change every 72 hours

**Comment by DeckOfPandas** [comment]
Thanks @pacharanero

Could you perhaps have a think about how to offer this in the UI? I suspect
it'll be clearer to have "every day" vs "not every day" in the first
instance, with options unfolding from those. Could you maybe draw it out,
or even make it...?

(Am very phoneable today, although I don't have very much voice)

**Comment by pacharanero** [comment]
I would agree that the default should be daily, with options for 'not daily'
Sorry no time to actually make it right now, I wouldn't know where to start in the codebase either.
Will call @DeckOfPandas this week (I'm working from home all week so should be a lot easier than the last few weeks)

![selection_011](https://cloud.githubusercontent.com/assets/2348385/11042538/9e1db1f6-870f-11e5-84c8-e80a7c3e5753.png)

Here's a quick sketch which would work for medication you take daily, or a variety of 'not-daily' options, where {number_of_days} is an integer. The thing it won't work for is when the interval varies, this is pretty unusual as a use-case, and I wouldn't worry about it for a version 1.0.
